app/forms/supplementary/DocumentsProduced.scala

```diff
@@ -80,4 +71,19 @@ object DocumentsProduced {
   )(DocumentsProduced.apply)(DocumentsProduced.unapply)
 
   def form(): Form[DocumentsProduced] = Form(mapping)
+
+  def fromString(value: String): DocumentsProduced = {
+    val fields = value.split('-')
+
+    if (fields.length == 0) DocumentsProduced(None, None, None, None, None, None)
+    else
+      DocumentsProduced(
+        Some(fields(0)),
+        Some(fields(1)),
+        Some(fields(2)),
+        Some(fields(3)),
+        Some(fields(4)),
```

Comment:
When you put only 3 fields probably you will have error here with retrieving 3, 4 or 5 field